src/docbuilder/queue.rs

```diff
@@ -84,4 +84,8 @@ impl DocBuilder {
 
         Ok(processed)
     }
+
+    pub fn run_git_gc(&self) {
+        self.index.run_git_gc();
+    }
```

Comment:
I don't think this is necessary, you can call `self.index.run_git_gc()` directly in the function.

Reply:
In which function?

If I'm replacing:
`doc_builder.run_git_gc()` with `doc_builder.index.run_git_gc()` it won't compile since `index` is private.

Reply:
Ah, I see. This seems fine then.